VocableUITests/Tests/TimingAndSensitivityTests.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  TimingAndSensitivityTests.swift
+//  VocableUITests
+//
+//  Created by Rhonda Oglesby on 4/18/22.
+//
+
+import XCTest
+
+class TimingAndSensitivity: BaseTest {
+    
+    func testHoverTimeIncreasesAndDecreases() {
+        settingsScreen.navigateToTimingAndSensitivityScreen()
+ 
+        // Verify default Hover Time value.
+        XCTAssertTrue(app.staticTexts[timingScreen.defaultHoverTime].exists)
+        
+        // Verify Hover Time can be decreased to the minimum value.
+        repeat {
+            timingScreen.decreaseHoverTime()
+            XCTAssertTrue(app.staticTexts[timingScreen.currentHoverTime].exists)
+            XCTAssertTrue(timingScreen.hoverTimeIncreaseButton.isEnabled)
+        } while !timingScreen.hasMinimumHoverTimeBeenReached()
+        
+        XCTAssertTrue(app.staticTexts[timingScreen.minimumHoverTime].exists)
+        XCTAssertTrue(!timingScreen.hoverTimeDecreaseButton.isEnabled)
+        
+        // Verify Hover Time can be increased to the maximum value.
+        repeat {
+            timingScreen.increaseHoverTime()
+            XCTAssertTrue(app.staticTexts[timingScreen.currentHoverTime].exists)
+            XCTAssertTrue(timingScreen.hoverTimeDecreaseButton.isEnabled)
+        } while !timingScreen.hasMaximumHoverTimeBeenReached()
+        
+        XCTAssertTrue(app.staticTexts[timingScreen.maximumHoverTime].exists)
+        XCTAssertTrue(!timingScreen.hoverTimeIncreaseButton.isEnabled)
```

Comment:
```suggestion
        XCTAssertFalse(timingScreen.hoverTimeIncreaseButton.isEnabled)
```
Same could apply up above 😄